hermes/.cargo/config.toml

```diff
@@ -28,6 +28,8 @@ rustflags = [
     "-C", "target-feature=-crt-static"
 ]
 
+[target.wasm32-unknown-unknown]
+rustflags = ["--cap-lints", "warn"]
```

Comment:
Shouldn't we remove this? This is temporary for the forked code. Once we properly format the forked code, these lines won't be needed anymore.

Reply:
🤔 thanks for catch, will try to remove it